src/api/libcellml/variable.h

```diff
@@ -39,6 +39,20 @@ class LIBCELLML_EXPORT Variable: public NamedEntity
     Variable& operator=(Variable n); /**< Assignment operator */
 
     /**
+     * @brief The INTERFACE_TYPES enum.
+     * Enum describing the interface types that are available for
+     * a given Variable.
+     */
+    enum INTERFACE_TYPES
+    {
+        INTERFACE_TYPE_UNDEFINED,
```

Comment:
It basically depends on whether you care if the library can round-trip a particular serialisation, or whether a semantically equivalent serialisation is fine. If the latter, then we probably don't need UNDEFINED. If the former, then we definitely do (and we probably will end up needing to care about whitespace etc which could get nasty!).